foodnet/membership/views.py

```diff
@@ -55,6 +56,42 @@ def invite(request):
     return render(request, 'foodnet/membership/invite.html', ctx)
 
 
+@login_required
+def departments_profiles(request, department_name=None):
+    # TODO: pagination of user profiles
+    assert department_name is not None
+    department = Department.objects.get(shortname=department_name)
```

Comment:
I wonder if it would look neater if you would use `get_objects_or_404` here as opposed to `assert`?
Do we still need this TODO?